app/models/pod.rb

```diff
@@ -1,11 +1,134 @@
 class Pod < ActiveRecord::Base
-  def self.find_or_create_by(opts) # Rename this method to not override an AR method
-    u = URI.parse(opts.fetch(:url))
-    pod = self.find_or_initialize_by(host: u.host)
-    unless pod.persisted?
-      pod.ssl = (u.scheme == 'https')? true : false
-      pod.save
-    end
-    pod
+  enum status: %i(
+    unchecked
+    no_errors
+    dns_failed
+    net_failed
+    ssl_failed
+    http_failed
+    version_failed
+    unknown_error
+  )
+
+  scope :reachable, lambda {
+    where.not(status: Pod.offline_statuses)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.